src/lib/db/NoteDb.ts

```diff
@@ -120,7 +121,7 @@ export default class NoteDb {
     await this.upsertFolder(parentPathname)
   }
 
-  async getAllDocsMap(): Promise<AllDocsMap> {
+  async getAllDocsMap(storageId: string): Promise<AllDocsMap> {
```

Comment:
I couldn't find any reasons to change this api like this. So I think we should revert this change.